data/transit/route/bus.json

```diff
@@ -6234,7 +6234,12 @@
       "locationSet": {"include": ["001"]},
       "tags": {
         "network": "Münchner Verkehrs- und Tarifverbund",
+        "network:guid": "DE-BY-MVV",
+        "network:wikidata": "Q259000",
+        "network:wikipedia": "en:Münchner_Verkehrs-_und_Tarifverbund",
```

Comment:
Should this rather link to the local (German) Wikipedia page?

Reply:
Yes, German stuff should link to the German wikipedia page, if there is one.

A few other things I noticed:
- make sure to update the `locationSet` to only include "de"
- replace the underscores with spaces in the wikipedia sitelinks